charts/stable/ser2sock/values.yaml

```diff
@@ -1,73 +1,55 @@
-# Default values for ser2sock.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-# -- Specifies the strategy used to replace old Pods by new ones
-strategyType: Recreate
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
 
 image:
-  # -- Image repository
   repository: tenstartups/ser2sock
-  # -- Image tag. Possible values listed [here](https://hub.docker.com/r/tenstartups/ser2sock/tags).
-  tag: latest
-  # -- Image pull policy
   pullPolicy: IfNotPresent
+  tag: latest
 
-nameOverride: ""
-fullnameOverride: ""
-
-# -- (string) Timezone the ser2sock instance should run as, e.g. 'America/New_York'
-timezone:
-# -- USB Device to use
-device: "/dev/ttyUSB0"
-# -- UID to run as
-puid: "1001"
-# -- GID to run as
-pgid: "1001"
-# -- Baudrate
-baudRate: 115200
-
-service:
-  # -- Kubernetes service type for the GUI
-  type: ClusterIP
-  # -- Kubernetes port where the GUI is exposed
-  port: 10000
-  # -- Service annotations for the GUI
-  annotations: {}
-  # -- (string) Cluster IP for the GUI
-  clusterIP:
-  # -- (string) External IPs for the GUI
-  externalIPs:
-  # -- (string) Loadbalancer IP for the GUI
-  loadBalancerIP:
-  # -- (string) Loadbalancer client IP restriction range for the GUI
-  loadBalancerSourceRanges:
-  # -- (string) Loadbalancer externalTrafficPolicy
-  externalTrafficPolicy:
-  # -- (string) nodePort to listen on for the GUI
-  nodePort:
-
-# -- CPU/Memory resource requests/limits or the GUI
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
-
-# -- Node labels for pod assignment of the GUI
-nodeSelector: {}
+strategy:
+  type: Recreate
 
-# -- Toleration labels for pod assignment of the GUI
-tolerations: []
+env:
+  # TZ:
+  # PUID:
+  # PGID:
```

Comment:
I'm OK with leaving these unset in the new version, but we had these specifically set before in the values to default to  `1001`.

Do we want to keep that default behavior, or leave it undefined, as a best practice?

Reply:
Usually we've been keeping them commented out as it really depends on the peoples setup.